extensions/amp-next-page/0.1/next-page-service.js

```diff
@@ -88,10 +88,7 @@ export class NextPageService {
     /** @private {?../../../src/service/viewport/viewport-impl.Viewport} */
     this.viewport_ = null;
 
-    /**
-     * @private
-     * {?../../../../src/service/position-observer/position-observer-impl.PositionObserver}
```

Comment:
Ooof.

Reply:
Interestingly, this completely breaks `gulp check-types`, which sees just the `@private`, and ignores the type name. A more widespread fix is coming up.